### ISSUE TITLE
Warn user when running without any `model__report` or `report__output` relationships

### ISSUE THREAD

**Issue by manuelma** [Spine-project/SpineOpt.jl#333]
Actually maybe we need to error in that case, since there's no point in running without being able to get results back.

**Comment by mihlema** [comment]
Yes I also think that there should be an error in this case.

**Comment by mihlema** [comment]
I think we touched upon this in one of our calls to possibly provide a default of all results and active cost terms being exported.
(I just recently ran a case study, while I forgot to add a certain cost term, and I had to reconstruct these cost terms)

I am not sure how to control this, i.e. how to provide a nice default for naive users and at the same time maintain the user flexibility to choose output/slimness. Maybe, if `model__report` and/or `output` and/or `output__report` and/or `report` are not defined everything gets exported - and we send the user a message that everything got exported - and if he wishes to customize go to the specific reaction in the documentation (@jkiviluo I don't think `report` etc are documented in the getting started yet)
